### DISCUSSION TITLE
Story 3 and 4 NFT art.

### DISCUSSION THREAD

**Discussion by DebravdH** [stellar/sorobanathon#115] [General]
We had some free time this week so we decided to work on art for story 3 and 4. The first link is to Rylee and my artistic drawing of the potions and fire of story 3. Rylee had her own paper, but decided it would be much better to colab with mom.. and I couldn't object seeing her artistic skills are just about as good as mine. lol

Erik kept saying he was drawing lotions (he meant potions) but he also decided to add a bit of a Christmas flare. You can see his on link 2. 

Link 3 is my personal favorite. This is an Erik Mommy colab where we replicate story 4 with legos and a drone. 😂 Erik named it Bone Kick. 

https://litemint.com/items/GD7GGRQ2SMPM72TXSWGK7AW7RIZM7BU3YNF24EUXBTYU5MV5W2GWC4VW/Potionsor
https://litemint.com/items/GB4DR5DT7KBTVYXG2Z6RCY7OVZMQUXNKB7AMOVR2V5WMW73VYI2SOWHC/lotions
https://litemint.com/items/GCPNIVYJQYRNRM7HKSZ7R6L4GXWKVZLIBW42NVCDW4AFKIWWVX4NPP7L/Bonekick

**Comment by stellarjohn** [comment]
Hi [DebravdH](https://github.com/DebravdH)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 900 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
